reference/api/keys.md

```diff
@@ -1,30 +1,313 @@
+| indexes.update   | Provides access to the [update index](/reference/api/indexes.md#update-an-index) endpoint. |
+| indexes.delete   | Provides access to the [delete index](/reference/api/indexes.md#delete-an-index) endpoint. |
+| tasks.get        | Provides access to the [get one task](/reference/api/tasks.md#get-task) and [get all tasks](/reference/api/tasks.md#get-all-tasks) endpoints. **Tasks from non-authorized `indexes` will be omitted from the response**. Also provides access to the [get one task by index](/reference/api/tasks.md#get-task-by-index) and [get all tasks by index](/reference/api/tasks.md#get-all-tasks-by-index) endpoints on authorized indexes. |
+| settings.get     | Provides access to the [get settings](/reference/api/settings.md#get-settings) endpoint and equivalents for all subroutes on authorized indexes. |
+| settings.update  | Provides access to the [update settings](/reference/api/settings.md#update-settings) and [reset settings](/reference/api/settings.md#reset-settings) endpoints and equivalents for all subroutes on authorized indexes. |
+| stats.get        | Provides access to the [get stats of an index](/reference/api/stats.md#get-stat-of-an-index) endpoint and the [get stats of all indexes](/reference/api/stats.md#get-stats-of-all-indexes) endpoint. For the latter, **non-authorized `indexes` are omitted from the response**. |
+| dumps.create     | Provides access to the [create dump](/reference/api/dump.md#create-a-dump) endpoint. **Not restricted by `indexes`.** |
+| dumps.get        | Provides access to the [get dump status](/reference/api/dump.md#get-dump-status) endpoint. **Not restricted by `indexes`.** |
+| version          | Provides access to the [get MeiliSearch version](/reference/api/version.md#get-version-of-meilisearch) endpoint. |
+
+#### `indexes`
+
+**Type:** array
+**Mandatory:** yes
+
+A list of indexes permitted for the key. `["*"]` for all indexes.
+
+#### `expiresAt`
+
+**Type:** string
+**Mandatory:** yes
+
+Date and time when the key will expire, represented in **ISO-8601** format. `null` if the key never expires.
+
+### Example
+
+<CodeSamples id="create_a_key_1" />
+
+#### Response: `201 Created`
+
+```json
+{
+    "description": "Add documents: Products API key",
+    "key": "d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4",
+    "actions": [
+        "documents.add"
+    ],
+    "indexes": [
+    "products"
+    ],
+    "expiresAt": "2021-11-13T00:00:00Z",
+    "createdAt": "2021-11-12T10:00:00Z",
+    "updatedAt": "2021-11-12T10:00:00Z"
+}
+```
+
+## Update a key
+
+<RouteHighlighter method="PATCH" route="/keys/:key"/>
+
+Update the description, permissions, or expiration date of an API key. To learn more about the variables sent in the body of the request, see the [create key](#body) endpoint.
+
+Updates to keys are **partial**. This means that you should provide only the fields you intend to update, as any fields not present in the payload will remain unchanged.
```

Comment:
Major nitpick: that `that` seems unnecessary. 

(Major as in extremely small and unimportant).

```suggestion
Updates to keys are **partial**. This means you should provide only the fields you intend to update, as any fields not present in the payload will remain unchanged.
```